ramls/declare-item-lost-request.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request body schema for declare item lost API",
+  "properties": {
+    "declaredLostDateTime": {
+      "description": "The date and time when the item to be declared lost",
+      "type": "string",
+      "format": "date-time"
+    },
+    "comment": {
+      "description": "Comment to the declare item lost action stored in the loan history",
```

That this comment is stored in the loan action history is specific to the implementation. Please could this description describe the intent of the comments. Maybe it is `stored` part of this statement that is the source of my discomfort.

```suggestion
      "description": "Comment explaining why the item has been declared lost",
```

---

This was taken from the similar functionality where comments were needed. I've changed it to lower the level of discomfort of anyone who reads it.